src/getBuildingId.py

```diff
@@ -0,0 +1,51 @@
+import json
+import boto3
+import decimal
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Dynamo resource"""
+dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
+buildingTable = dynamodb.Table('Buildings')
+
+"""Workaround for formatting issue
+   Source: http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Python.04.html
+"""
+class DecimalEncoder(json.JSONEncoder):
```

Comment:
Pull this class out into a separate file, then import it here and in the other lambda functions so we can reduce copied code.